CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contributing to Obl.ong
+
+First of all, thank you for wanting to contribute to Obl.ong! Everyone is welcome to help and encouraged to help.
+
+## 📜 Code of Conduct
+
+Obl.ong is governed by the [Code of Conduct](https://github.com/obl-ong/code-of-conduct). You are expected to follow it. Basically, treat other humans with respect and don't discriminate based on any characteristic. You can report any violation to team@obl.ong, and we will take care of it ASAP.
+
+## 🪳 Bug Reporting
+
+Bug Reports are vital to development. You can report bugs using GitHub issues. However, we need information to help squash the bug. Basically:
+
+- Ensure you are using the latest commit or version of Obl.ong.
+- Make sure the issue hasn't already been posted by searching the issues.
+- Give information about the bug: OS, OS version, Browser Version (if it occured in the browser), ruby version (`ruby -v`), git commit (`git rev-parse --short HEAD`), screenshot (if needed).
+
+Also, please do **not** submit security issues via GitHub issues. Please send an E-mail to team@obl.ong.
+
+## 💻 Setting up a development environment
+
+1. Install the currently supported version Ruby. You can use a tool like [Ruby Version Manager (RVM)](https://rvm.io/) to install ruby. Check the `.ruby-version` file to find it. Usually, it's the latest version, but it never hurts to check.
+2. Install Rails. A lot of times it may come preinstalled. Run `rails -v` to check.
+3. Install [bun.sh](https://bun.sh). You can do this by running `curl -fsSL https://bun.sh/install | bash
+4. Pull the submodules. Run `git submoudle init && git submodule update`
```

Comment:
git submodule is spelled wrong

Reply:
Fixed in [9ccf2e8](https://github.com/obl-ong/admin/pull/148/commits/9ccf2e8544214ed40fd8ac0d73b6800384d3b49f)